UM/Scene/Selection.py

```diff
@@ -71,23 +71,30 @@ def getSelectionCenter(cls):
     #   \param operation \type{Class} The operation to create and push. It should take a SceneNode as first positional parameter.
     #   \param args The additional positional arguments passed along to the operation constructor.
     #   \param kwargs The additional keyword arguements that will be passed along to the operation constructor.
+    #
+    #   \return list of instantiated operations
```

Comment:
Adds a list of instantiated operations, so the method calling applyOperations can access each individual operation (eg show progress)